Detectors/TRD/reconstruction/src/TrackletsParser.cxx

```diff
@@ -66,6 +66,15 @@ int TrackletsParser::Parse(std::array<uint32_t, o2::trd::constants::HBFBUFFERMAX
   mIgnoreTrackletHCHeader = options[TRDIgnoreTrackletHCHeaderBit];
   mEventRecord = eventrecord;
   mEventRecords = eventrecords;
+  //A HACK !!!
+  /*if(mFEEID.supermodule>15){
+    mHeaderVerbose=true;
+    LOG(info) << "Verbose output as we have detector : " << detector;
+  }
+  else {
+    mHeaderVerbose=options[TRDHeaderVerboseBit];
+  }*/
+  //END HACK !!!
```

Comment:
Is the hack supposed to stay in?

Reply:
ja its for debugging the last 2 supermodules, but i can take it out if it offends others.

Reply:
no worries, just double-checking if that was a leftover or it is still used. Fine with me to keep it in for now

Reply:
removed ;-)